.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-  - "3.4"
```

Comment:
What stops 3.4 from working? I'm not too concerned about dropping support for it in the contrib scripts since it's EOL and distros still shipping it have a clear upgrade path to newer distros, but if it's something minor we might as well fix it.

Reply:
`subprocess.run` is [3.5+](https://docs.python.org/3/library/subprocess.html#subprocess.run]). I've used it to manage making external shell commands (`git` etc). You mentioned in #1540 that 3.5 was ok as a minimum - is this still the case? :smile:

Reply:
I think for contrib scripts, 3.5 is ok as a minimum. CentOS 6 still has 3.4 as the newest it can get, but I'm inclined to not really care about CentOS 6 users.

Reply:
Ah I see, I guess that depends how many users it would affect 🤔 Do you know how many users are on CentOS 6?

Reply:
I count about 70 unique IPs accessing the CentOS 6 repos in about 2 days worth of logs, but since this only affects contrib scripts which are rarely used it's practically negligible.

Admittedly, there's more CentOS 6 users than CentOS 7 users, apparently. But if they want to use the contrib scripts, they should upgrade.

Reply:
How interesting! CentOS 6 is eol in [November 2020](https://wiki.centos.org/About/Product) which coincides reasonably well with Python 2 eol anyway so there's some incentive to upgrade there anyway I guess. As long as the 3.5+ requirement is made clear in the documentation/help of the scripts then it should be fine? Is it worth me adding back the 3.4 environment in travis but excluding the contrib tests from it? If overviewer itself still supports 3.4 it's probably good to have the tests running for it.

Reply:
Disabling certain tests for certain Python versions introduces more complexity than I'd like, I think it's fine if our position on this is just "Overviewer may work with it, but if it doesn't then you should upgrade."

There really isn't any reason for them not to upgrade.

Reply:
That's a good point, fragmenting tests across versions is not a great idea now I think about it 😅 Sounds good, hopefully it'll help give people a nudge towards upgrading 🤞